ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ClusterDetailsTable.tsx

```diff
@@ -146,10 +139,10 @@ function ClusterDetailsTable({
                                     <Tr key={checkName}>
                                         <Td dataLabel="Check">
                                             <Link
-                                                to={generatePath(coverageCheckDetailsPath, {
+                                                to={`${generatePath(coverageCheckDetailsPath, {
```

Comment:
Although not a blocker to merge, let’s start to replace `generatePath` with the following:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/utils/urlUtils.ts#L40-L55